python/ray/train/_internal/backend_executor.py

```diff
@@ -547,6 +548,7 @@ def initialize_session(
                 controller_actor_id=core_context.get_actor_id(),
                 datasets=datasets,
                 worker_group=self.worker_group,
+                start_time=time.time(),
```

Comment:
As a user, what time do we expect this to be? Should this `time.time()` call be called here, during initialization of the `Trainer`, or somewhere else?

Reply:
That's a good question! The `start_time` here will be used to sort the train runs in the dashboard table, and move the latest ones to the top. Currently we set the `start_time` after initialize session and before training starts, which should suffice this use case.

For more accurate timestamp, I think we need to build a event system and report the Ray Train internal events with it.

Reply:
Is it exposed to the user in a column attribute?

Reply:
yes, it will be exposed.

Reply:
I think it's probably more explainable if you just have it be the timestamp at which the `BackendExecutor` is initialized (i.e. in `BackendExecutor.__init__`). What do you think?

Reply:
Yeah that also makes sense. I can set the time there!